src/gui/tray/TalkReply.qml

```diff
@@ -0,0 +1,84 @@
+import QtQuick 2.15
+import Style 1.0
+import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.2
+import com.nextcloud.desktopclient 1.0
+
+ColumnLayout {
+    id: talkLayout
+    spacing: 0
+    
+    Layout.fillWidth: true        
+    Layout.fillHeight: true        
+    Layout.minimumWidth: 170
+    Layout.minimumHeight: parent.height
+    
+    function sendMessage() {        
+        if (talkMessage.text === "") {
+            rectangleInput.border.color = Style.errorBoxBorderColor;
+            return;
+        }
+        rectangleInput.border.color = Style.menuBorder;
+        UserModel.currentUser.sendChatMessage(model.conversationToken, talkMessage.text, model.messageId);
+        talkMessageSent.text = talkMessage.text;
+        talkMessage.clear();
+        rectangleInput.visible = false
+    }
+    
+    Text {
+        id: talkMessageSent
+        color: Style.menuBorder
+        visible: talkMessageSent.text !== ""
+    }
+    
+    Rectangle {  
+        id: rectangleInput
+        width: 180
+        height: 24
+        border.width: 1
+        border.color: Style.menuBorder
+        radius: 22
+        clip: true
+        TextInput {
+            id: talkMessage
+            font.pixelSize: Style.topLinePixelSize
+            height: activityTextInfo.height
+            width: parent.width
```

Comment:
@camilasan @mgallien @claucambra My QML knowledge might be failing me, but, shouldn't we then use `anchors.left: 
 parent`, `anchors.right: parent` since we are using `anchors.verticalCenter` and, should no use `TextInput.width` in this case?